modules/reference/pages/command/server-stop.adoc

```diff
@@ -13,12 +13,7 @@
 
 The `server stop` command stops the named Open Liberty server.
 
-This command triggers the server to enter a quiesce phase before stopping. The quiesce phase prevents new application requests from entering the server and allows existing requests time to complete. After that time, the server shuts down and the server process exits.
-
-//* Equivalent command for Maven.
-//* Equivalent command for Gradle.
-* xref:command/server-run.adoc[Start an Open Liberty server in the foreground.]
-* xref:command/server-start.adoc[Start an Open Liberty server in the background.]
+Normal server stop includes a quiesce stage before the server is shut down. The quiesce stage, a period of 30 seconds, allows for services to perform pre-shutdown work, for example, stopping inbound listeners but allowing existing requests to complete. Applying the `--force` option to the stop command skips the quiesce stage. The `--force` option has no effect if server stop was already invoked. If you use the `--force` option, you might see unexpected exceptions in the `messages.log` file that occur after the server received the server stop command.
```

Comment:
## Review 

If you use the `--force` option, you might see unexpected exceptions in the `messages.log` file that occur after the server received the server stop command.
-->
If you use the `--force` option, you might see unexpected exceptions in the `messages.log` file that occurs after the server received the server stop command.
___
**Perform** being flaffed by Acrolinx.
-->
 The quiesce stage, a period of 30 seconds, allows services with the pre-shutdown work
( is pre-shutdown ok or pre shutdown without hyphenation?
___
invoked also getting flagged
-->
The `--force` option has no effect if server stop was already initiated.

Reply:
If you use the --force option, you might see unexpected exceptions in the messages.log file that occurs after the server received the server stop command.

the subject and verb dont agree in this suggestion (you might see exceptions...occurs after..)

revised to address other comments- thanks